WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift

```diff
@@ -484,6 +492,11 @@ private extension AppSettingsViewController {
             action: pushDebugMenu()
         )
 
+        let designSystem = NavigationItemRow(
+            title: NSLocalizedString("Design System", comment: "Navigates to design system gallery only available in development builds"),
```

Comment:
@alpavanoglu  Just a manual reminder to use reverse-DNS keys for localized strings.

[I noticed this while reviewing the 22.3 code freeze PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/20626)